### PR TITLE
[REVIEW] Add nightly benchmarks [skip-ci-changelog]

### PR DESCRIPTION
**Pull request by mnicely**
### Info
This PR does the following:
1. Add nightly benchmarks using pytest to cuSignal
2. Add CC 8.6 for CUDA 11.1 builds
3. Replaces `logger` with `gpuci_logger`

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by mnicely** [comment]
@dillon-cullinan @rlratzel 
I think this is working correctly from my local docker tests.

File: ci/docs/build.sh [hunk 1]
```diff
@@ -18,27 +18,27 @@ export LIBCUDF_KERNEL_CACHE_PATH="$HOME/.jitify-cache"
...
-logger "Check GPU usage..."
+gpuci_logger "Check GPU usage..."
 nvidia-smi
 
-logger "Activate conda env..."
+gpuci_logger "Activate conda env..."
 source activate rapids
 # TODO: Move installs to docs-build-env meta package
 conda install -c anaconda markdown beautifulsoup4 jq
 pip install sphinx-markdown-tables
```
Comment (ajschmidt8) [inline ci/docs/build.sh:31]:
```suggestion
# TODO: Move installs to docs-build-env meta package
conda install -c anaconda markdown beautifulsoup4 jq
pip install sphinx-markdown-tables
```

similarly to the comment on Mike's PR below, this whole `TODO` block can be removed. I see `sphinx-markdown-tables`, `markdown` and `beautifulsoup4` in the [rapids-doc-env](https://github.com/rapidsai/integration/blob/branch-0.16/conda/recipes/rapids-doc-env/meta.yaml#L37-L49) and `jq` in our [gpuci devel images](https://github.com/rapidsai/gpuci-build-environment/blob/branch-0.16/rapidsai/devel.Dockerfile#L80-L83).

https://github.com/rapidsai/cudf/pull/6329#discussion_r506506128

**Review by ajschmidt8 — COMMENTED** [review]
Requested 1 housekeeping change that wasn't a part of your changes.

File: ci/docs/build.sh (hunk 1 above)
Comment (mnicely) [reply ci/docs/build.sh:31]: Done!

**Comment by mnicely** [comment]
rerun tests

**Review by BradReesWork — APPROVED** [review]

**Comment by BradReesWork** [comment]
rerun tests

**Pull request merged**